ui/src/FeedbackButton.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import FeedbackIcon from "@material-ui/icons/FeedbackOutlined";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles"
+import { ScholarReaderContext, PaperId } from "./state";
+import queryString from "querystring";
+
+function mkFeedbackLink(paperId?: PaperId, extraContext?: Object) {
+    console.log(extraContext);
+    const baseUrl = "https://docs.google.com/forms/d/e/1FAIpQLSdnTn4ng-3SsNqwr6M7yF54IhABNAw9_KIjPdWC746fIe546w/viewform";
+    const params = {
+        // This ID is produced by Google Forms. If the form changes we
+        // may need to change this value
+        "entry.331961046": JSON.stringify(Object.assign(
+            {},
+            extraContext || {},
+            { paperId }
+        ))
+    };
+    return `${baseUrl}?${queryString.stringify(params)}`;
+}
+
+const useStyles = makeStyles({
```

Comment:
Generally I've been trying to use Less rather than styled components as I think it makes live-reloading easier. Do you think we should consider using styled components?

Reply:
Ah, interesting!

I went this route as it was `@material-ui/`s canonical mechanism for styling.  That said I'll go the `less` route, given it's more consistent with the existing code-base and works better for live-reload purposes!

Reply:
Fixed. I removed `makeStyles` and ported the styles into less.